doc/scapy/build_dissect.rst

```diff
@@ -2,7 +2,7 @@
 Adding new protocols
 ********************
 
-Adding new protocol (or more correctly: a new *layer*) in Scapy is very easy. All the magic is in the fields. If the 
+Adding a new protocol (or more correctly: a new *layer*) in Scapy is very easy. All the magic is in the fields. If the 
```

Comment:
An alternative would be to use plural form: "Adding new protocols".

Reply:
(OTOH then `a new *layer*` would have to be changed to plural as well. So better just leave it as it is.)